network_simulator/flow.py

```diff
@@ -4,9 +4,13 @@
 """
 
 from abc import ABCMeta, abstractmethod
```

Comment:
Add some brief comments at the beginning of each class that describe what the Event does.